### DISCUSSION TITLE
A Strange Error.

### DISCUSSION THREAD

**Discussion by onurjkk** [msiemens/tinydb#426] [Q&A] [answered]
Hello, i tried to look for something in the database to see if it existed, but if it already exists it gives me the error that you can see in the image below; I think it's a module error inside the tables, but I don't know.
![code](https://user-images.githubusercontent.com/76625957/131045940-7dc9394c-3900-4dad-929a-bd1a989d9273.png)
![error](https://user-images.githubusercontent.com/76625957/131045941-d47b0d8a-86cc-4f27-b27f-8e81d742f830.jpg)

**Comment by eugene-eeo** [comment, marked as answer]
The error happens in the

```
result = inventory.search(...)
```

You're passing a boolean to `inventory.search(...)`, but [`Table.search`](https://tinydb.readthedocs.io/en/latest/api.html#tinydb.table.Table.search) expects a Query object, or a callable instead.
Most likely, you wanted to do something like (Note the _uppercased_ "Inventory"):

```
Inventory = Query()
result = inventory.search(Inventory.name == "common_circle")
```

**Reply by onurjkk** [reply]
thanks man, you're great!
